policyengine_us/parameters/gov/states/hi/tax/income/credits/food_excise_tax/minor_child/support_proportion_threshold.yaml

```diff
@@ -0,0 +1,14 @@
+description: Hawaii provides an additional food/excise credit amount for each minor child receiving more than this proportion of support from public agency.
+values:
+  2021-01-01: 0.5
+metadata:
+  unit: /1
+  label: Hawaii food/excise tax credit minor child public agency support's proportion threshold
+  reference:
+    - title: Hawaii Income Tax Law, Chapter 235, Section 235-55.85, (c)
```

Comment:
does the law mention the 0.5 rate? if not comment that "# Law does not specify the public support rate"